src/test/java/org/takes/http/FtRemoteTest.java

```diff
@@ -52,10 +52,11 @@
  * @version $Id$
  * @since 0.21
  *
- * @todo #730:30min Fix FtRemote for empty response body test case and
- *  unignore returnsAnEmptyResponseBody. After that the
- *  {@link org.takes.tk.TkSlf4jRemoteTest} should be fixed and relevant
- *  test case should be unignored.
+ * @todo #800:30min We can't assert if FtRemote returns an emptybody because
+ *  some bug in jcabi-http (reported in
+ *  https://github.com/jcabi/jcabi-http/issues/177). After this bug is fixed,
+ *  correct the testing for this condition in returnsAnEmptyResponseBody and
+ *  TkSlf4jRemoteTest.
```

Comment:
@paulodamaso Please preserve the
> After that the
 *  {@link org.takes.tk.TkSlf4jRemoteTest} should be fixed and relevant
 *  test case should be unignored.

from original puzzle as otherwise the original test case will be forgotten. Also I don't think that bug in `jcabi-http` is what we have. Have you tried it with different http client? Also does it will fix the original issue #712?